checker/src/types/mod.rs

```diff
@@ -81,7 +81,7 @@ impl TypeId {
 
 	/// TODO remove. Shortcut for inferred this
 	pub const ANY_INFERRED_FREE_THIS: Self = Self(19);
-	/// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new.target
+	/// <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new.target>
```

Comment:
This error is raised by [doc_markdown](https://rust-lang.github.io/rust-clippy/master/index.html#/doc_markdown), so I believe it's because it converts to a markdown link

Reply:
Ah yes it's part of markdown: https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses